tests/common/fixtures/conn_graph_facts.py

```diff
@@ -31,3 +31,10 @@ def conn_graph_facts(duthost, testbed_devices):
                 conn_graph_facts = localhost.conn_graph_facts(host=duthost.hostname, filename=lab_conn_graph_file)['ansible_facts']
                 return conn_graph_facts
     return conn_graph_facts
+
+@pytest.fixture(scope="module")
+def fanout_graph_facts(testbed_devices, conn_graph_facts):
+    localhost = testbed_devices["localhost"]
+    fanout_host = conn_graph_facts["device_conn"]["Ethernet0"]["peerdevice"]
+    facts = localhost.conn_graph_facts(host=fanout_host, filename=LAB_CONNECTION_GRAPH)["ansible_facts"]
```

Comment:
what is LAB_CONNECTION_GRAPH? Please use the format specified in 'conn_graph_facts' to obtain the correct graph file from the inv mapping so that it can be used by all

Reply:
LAB_CONNECTION_GRAPH - missed during merge, it should be direct path to the inventory file.

If to use format like in "conn_graph_facts" fixture, this fixture code should be udpated, to have separate fixture or what that returns current inventory used, then this inventory can be used anywhere. Do you agree?

Reply:
hardcoding this value will not allow others to make use of this fixture when their connection graph is something other than 'lab_connection_graph.xml'. you need to pass in the correct graph file to 'filename' based on the inventory name the user specifies.

Reply:
Fixed